src/MenuBar.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styles from '../public/css/index.css';
+import { Nav, Navbar, NavItem, MenuItem, DropdownButton } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { setUserAction } from './redux/userActions.js';
+import axios from 'axios';
+
+export class homeMenuBar extends React.Component {
+  constructor(props) {
+    super(props);
+
+  }
+
+  componentDidMount() {
+    var self = this;
+    axios.get('/user')
+          .then(function (response) {
+                  if (response.data !== '') {
+                    console.log(response.data);
+                    self.props.setUser(response.data);
+                  }
+          })
+          .catch(function (error) {
+              console.log(error);
+          });
+  }
+
+  logout() {
+    axios.get('/logout').then(function(response) {
+        console.log(response);
+        if (response.data === true) {
+            document.location.href='/';
+
+        }
+    }).catch(function(error) {
+        console.log(error);
+    });
+  }
+
+
+
+  render() {
+    if (this.props.homeordash === "home") {
+    return (
+      <Navbar collapseOnSelect className={styles.navigationbar}>
+        <Navbar.Header>
+          <Navbar.Brand className={styles.navbarheader}>
+            <a href="/home/about">Midtown Assistance Center</a>
+          </Navbar.Brand>
+          <Navbar.Toggle />
+        </Navbar.Header>
```

Comment:
You should break these out into two different components for "dash" and "home" navbars.